themes/vue/layout/partials/language_dropdown.ejs

```diff
@@ -5,7 +5,7 @@
     <li><a href="https://jp.vuejs.org/<%- page.path %>" class="nav-link" target="_blank">日本語</a></li>
     <li><a href="https://ru.vuejs.org/<%- page.path %>" class="nav-link" target="_blank">Русский</a></li>
     <li><a href="https://kr.vuejs.org/<%- page.path %>" class="nav-link" target="_blank">한국어</a></li>
-    <li><a href="https://br.vuejs.org/<%- page.path %>" class="nav-link" target="_blank">Português</a></li>
```

Comment:
The official language is [Portugues](https://en.wikipedia.org/wiki/Brazil) so this specification is meaningless

Reply:
@ludo237 American English is the same as England English? It's not meaningless.

Reply:
I didn't want to be mean but it's not suitable to create a menu that groups every dialectical differences. Generally speaking the only two languages that have this treatment are English (UK and US) and Chinese (simplified and standard)

Reply:
It's not meaningless, but I wouldn't say it's too necessary either. Similar to English, you don't specify "American English" in a dropdown _unless_ you have both American and British English.

Reply:
We Portuguese/Brazilians are used to diferenciate our native Portuguese languages, that's why i did the PR. 

Just to make clear that it's Brazilian Portuguese, nothing more nothing less. :) 

As it seems this is not an issue i can live with it as it is.